### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by brandmaier**
Here is a shortened and revised version

### CODE REVIEW

File: r-references.bib [hunk 1]
```diff
@@ -104,3 +104,40 @@ @article{barker2022introducing
...
+    pages = {5--10},
+    url = {https://journal.r-project.org/archive/2011-1/RJournal_2011-1_Wickham.pdf},
+  }
+
+@article{bates2014fitting,
+  title={Fitting linear mixed-effects models using lme4},
+  author={Bates, Douglas and M{\"a}chler, Martin and Bolker, Ben and Walker, Steve},
+  journal={arXiv preprint arXiv:1406.5823},
+  year={2014}
+}@Manual{R-koRpus,
```
Comment (aaronpeikert) [inline r-references.bib]:
```suggestion
}
@Manual{R-koRpus,
```

Comment (aaronpeikert) [inline r-references.bib]:
```suggestion
}
@Manual{R-koRpus,
```

File: manuscript.Rmd [hunk 2]
```diff
@@ -76,95 +90,99 @@ We would like to respond with a quote from the Science Code Manifesto (http://sc
...
-While we highly encourage this from both a software engineering perspective and for scientific progress, it poses further difficulties to citation-based metrics.
-This especially punishes highly relied-on backends such as optimization algorithms: For example, the NLopt optimization suite is cited 1,711 times on google scholar.
-However, the algorithms from that suite are used in countless software packages, for example, as a backend of OpenMx which again has 811 citations.
-Moreover, the algorithms from that suite are all open source, and therefore serve as a reference for others.
-As an example, consider the lme4 package, that adapted one of the algortihms from NLopt.
-This package has 57,820 citations on google scholar, yet no single lme4 user probably ever cited the NLopt library.
-We therefore argue that merrit statements are much more useful to assess impact of research software.
+## ID 5: Date of first fully functional public version
+
+The date of the first fully functional public version serves as a denominator in computing the number of citations (see ID 9) relative to software age.
```
Comment (aaronpeikert) [inline manuscript.Rmd:115]: Shouldn't we heavily criticize building an ad hoc H-Index, the very reason why DORA was proposed?

File: manuscript.Rmd [hunk 3]
```diff
@@ -76,95 +90,99 @@ We would like to respond with a quote from the Science Code Manifesto (http://sc
...
 ## ID 11: GitHub Stars
 
 We object to the use of Github stars as a general metric.
-First of all, it is tied only to a single, commercial platform and allows no freedom to choose free alternatives such as local Gitlab instances or other platforms.
-More importantly, we question the validity of such metrics that like rating systems on commercial platforms are too easy to be gamed by creating fake accounts that provide high ratings.
-
-## ID 6: Date of most recent substantive update
-<!-- see issue #7 -->
+First of all, it is tied only to a single, commercial platform.
+More importantly, we question the validity of such metrics that, like rating systems on commercial platforms, are too easy to be gamed.
```
Comment (aaronpeikert) [inline manuscript.Rmd]: I don't agree. Gaming the system is always easy. I'd rather criticize that software has very different ways of making an impact, therefore the merit statement. A backend library might not bost with many stars but many dependencies. Stars may be one way of showing the impact but can't be generally accepted.

File: manuscript.Rmd [hunk 4]
```diff
@@ -76,95 +90,99 @@ We would like to respond with a quote from the Science Code Manifesto (http://sc
...
+Broader impact in science is more likely with open source software.
 In addition, many aspects of rigor are not possible to evaluate for closed-source software --- for example, whether it is well-tested or whether bugs have been fixed.
-We therefore propose to either score the license directly, giving points to open-source licenses as defined by the Open Source Society (https://opensource.org/licenses), or to assess the indicators of rigor as discussed in the second part of this commentary, where it is only possible to gain points for certain aspects of rigor if the software is open-source.
+Therefore, we propose to penalize software if it does not adhere to an open-source licenses (those approved in a review process by the Open Source Society (https://opensource.org/licenses) by allowing them only half of the achievable total points. <!--Alternatively, one may indirectly score openness by assessing the indicators of rigor as discussed in the second part of this commentary, where it is only possible to gain points for certain aspects of rigor if the software is open-source.-->
```
Comment (aaronpeikert) [inline manuscript.Rmd]: Yes OSI approved licences should count!

File: manuscript.Rmd [hunk 5]
```diff
@@ -185,48 +206,39 @@ Another metric that could be assessed is code coverage, defined as the percentag
...
+We believe this is a fair assessment of the real impact no matter whether someone contributed documentation to run certain types of models, whether someone contributed a specific feature that enabled a given research project to use the software, or whether someone is the primary author of the entire package.
+
 
 # Summary
 
 In sum, we are thankful to  @gartner2022responsible for highlighting the importance of research software contributions as scientifically valuable products.
-Specifically, we value the distinction of different classes of software projects (e.g., small scripts vs large-scale team efforts), ....
-However, we are critical of the proposed quantitative impact measures.
-For the reasons mentioned here, We propose to emphasize that qualitative statement on impact, in which software authors should argue in how far their project paved the way for other scientific endeavours.
+We believe that the current proposal must aim to better reflect the distinction between rigor and impact.
```
Comment (aaronpeikert) [inline manuscript.Rmd]:
```suggestion
We believe that the current proposal must aim to better reflect the distinction between rigor and impact for software like similar to what was proposed to evaluate journal articles.
```

File: manuscript.Rmd (hunk 3 above)
Comment (brandmaier) [reply manuscript.Rmd]: do you recommend to entirely drop the criticism of the stars?

Comment (aaronpeikert) [reply manuscript.Rmd]: If we still have space issues yes, I'd move it into l. 226 together with download numbers, etc.

Comment (brandmaier) [reply manuscript.Rmd]: good idea!

File: manuscript.Rmd (hunk 2 above)
Comment (brandmaier) [reply manuscript.Rmd:115]: yes, thanks for reminding me. Done!

**Pull request merged**